cmd/misc/main.go

```diff
@@ -149,6 +149,42 @@ func main() {
 		for epoch := opts.StartEpoch; epoch <= opts.EndEpoch; epoch++ {
 			err = exporter.ExportEpoch(epoch, rpcClient)
 
+			if err != nil {
+				logrus.Errorf("error exporting epoch: %v", err)
+			}
+			logrus.Printf("finished export for epoch %v", epoch)
+		}
+	case "export-epoch-missed-slots":
+		logrus.Infof("exporting epochs with missed slots")
+		epochs := []uint64{}
+		err = db.ReaderDb.Select(&epochs, `
+			WITH last_exported_epoch AS (
+				SELECT (MAX(epoch)*$1) AS slot 
+				FROM epochs 
+				WHERE finalized 
+				AND rewards_exported
+			)
+			SELECT e.epoch FROM (
+				SELECT (slot / $1) AS epoch
+				FROM blocks
+				WHERE status = '0' 
+				AND slot < (SELECT slot FROM last_exported_epoch)
+				ORDER BY slot desc
+			) AS e 
+			GROUP BY e.epoch 
+			ORDER BY epoch;
```

Comment:
```suggestion
			WITH last_exported_epoch AS (
			    SELECT (MAX(epoch) * $1) AS slot
			    FROM epochs
			    WHERE finalized AND rewards_exported
			)
			SELECT epoch
			FROM blocks
			WHERE status = '0' AND slot < (SELECT slot FROM last_exported_epoch)
			GROUP BY epoch
			ORDER BY epoch;
```

Reply:
you are right the query was more complex then needed - simplified now.